src/components/dataSet-settings.js

```diff
@@ -234,11 +227,10 @@ class DataSetSettings extends React.Component {
                     .periodDSDBTrimming,
             }
 
-            const sumaryString = this.state.specificSetting.periodDSDownload
             const sumarySettings =
                 this.state.specificSetting.periodDSDownload === undefined
                     ? undefined
-                    : this.arrangeWord(sumaryString)
+                    : this.state.specificSetting.periodDSDownload
```

Comment:
This also looks like a functional change?

Reply:
I'll keep the lines